content/learn/05.communication/10.can/can.md

```diff
@@ -208,6 +208,6 @@ To add termination resistors, connect a 120-ohm resistor between the CAN_H and C
 
 Below you can find an overview of tutorials we have created for our different products.
 
-- [Arduino UNO R4 WiFi CAN Bus](tutorials/uno-r4-wifi/can/)
+- [Arduino UNO R4 WiFi CAN Bus](https://docs.arduino.cc/tutorials/uno-r4-wifi/can/)
```

Comment:
Hi @n1lby73 , thank you for the submission. The link is indeed wrong, but I'm updating it to use a relative link instead. The current one is missing the first `/`, hence it is not working.

```suggestion
- [Arduino UNO R4 WiFi CAN Bus](/tutorials/uno-r4-wifi/can/)
```